code/controllers/configuration/entries/general.dm

```diff
@@ -399,6 +399,8 @@
 /datum/config_entry/string/chat_announce_new_game
 	config_entry_value = null
 
+/datum/config_entry/string/service_command//sunset discord bot

```

Comment:
you can just stick this in like `sunset/code/controllers/configuration/entries/general.dm`

Reply:
i know, just being quick, planning on modularizing later